src/Thoth.Json.WebApi/Thoth.Json.WebApi.fs

```diff
@@ -0,0 +1,18 @@
+namespace Thoth.Json.WebApi
```

Comment:
I don't think this belongs to Thoth.Json.Net

You are adding `Microsoft.AspNet.WebApi` as a dependencies of Thoth.Json.Net which is not needed. It should be in the future a separate package like `Thoth.Json.Giraffe`

Reply:
This is the _new_ package Thoth.Json.WepApi.

Reply:
Got it. Thoth.Json,Giraffe is now also in its own repository :-)